### DISCUSSION TITLE
Default values for BelongsTo relations are shown on index/detail with an empty link

### DISCUSSION THREAD

**Discussion by sedlatschek** [laravel/nova-issues#3419] [Q&A] [answered]
- Laravel Version: 8.44.0
- Nova Version: 3.26.1
- PHP Version: 7.4.6
- Database Driver & Version: MySQL 5.7
- Operating System and Version: Win10
- Browser type and version: Chrome 88

### Description:
There is a problem when having a nullable BelongsTo field with a default value. Even when the value of the field is null, the index and detail page show some value with an empty link.

```php
// example field
BelongsTo::make('Organisation')->default(1) ->nullable(),
```

The value of `organisation_id` is null.
![image](https://user-images.githubusercontent.com/58665536/120489380-df6fa700-c3b7-11eb-9ff4-92d207fa2165.png)

Yet on index and detail "1" is displayed with a link to /nova.
![image](https://user-images.githubusercontent.com/58665536/120489496-f8785800-c3b7-11eb-8d21-84b15ad2593d.png)
![image](https://user-images.githubusercontent.com/58665536/120489601-0cbc5500-c3b8-11eb-8f97-ddc216363ecf.png)

### Detailed steps to reproduce the issue on a fresh Nova installation:
1. Create two models
2. Create a nullable belongsTo relation from one to the other
3. Set some default value in the BelongsTo nova field
4. Create a resource with the value null for the BelongsTo field
5. Go to resource index -> The BelongsTo field is not empty
6. Display a resource -> The BelongsTo field is not empty

**Comment by crynobone** [comment, marked as answer]
Yes, that's the current behavior for default. For such a requirement, you mostly just want a default value on creating. 

https://crynobone.com/add-field-default-only-on-create-on-laravel-nova/

There a PR merged to add this for next major release.
